flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/UnsignedTypeConversionITCase.java

```diff
@@ -229,27 +180,84 @@ private void prepareData() {
         tEnv.createTemporaryView("data", dataTable);
     }
 
-    @After
-    public void cleanup() {
-        stopDb();
+    @Test
+    public void testUnsignedType() throws Exception {
+        // write data to db
+        tEnv.executeSql(
+                        "insert into jdbc_sink select"
+                                + " tiny_c,"
+                                + " tiny_un_c,"
+                                + " small_c,"
+                                + " small_un_c ,"
+                                + " int_c,"
+                                + " int_un_c,"
+                                + " big_c ,"
+                                + " big_un_c from data")
+                .await();
+
+        // read data from db using jdbc connection and compare
+        PreparedStatement query =
+                connection.prepareStatement(
+                        String.format(
+                                "select tiny_c, tiny_un_c, small_c, small_un_c,"
+                                        + " int_c, int_un_c, big_c, big_un_c from %s",
+                                TABLE_NAME));
+        ResultSet resultSet = query.executeQuery();
+        while (resultSet.next()) {
+            assertEquals(127, resultSet.getObject("tiny_c"));
+            assertEquals(255, resultSet.getObject("tiny_un_c"));
+            assertEquals(32767, resultSet.getObject("small_c"));
+            assertEquals(65535, resultSet.getObject("small_un_c"));
+            assertEquals(2147483647, resultSet.getObject("int_c"));
+            assertEquals(4294967295L, resultSet.getObject("int_un_c"));
+            assertEquals(9223372036854775807L, resultSet.getObject("big_c"));
+            assertEquals(new BigInteger("18446744073709551615"), resultSet.getObject("big_un_c"));
+        }
+
+        // read data from db using flink and compare
+        Iterator<Row> collected =
+                tEnv.executeSql(
+                                "select tiny_c, tiny_un_c, small_c, small_un_c,"
+                                        + " int_c, int_un_c, big_c, big_un_c from jdbc_source")
+                        .collect();
+        List<Row> result = CollectionUtil.iteratorToList(collected);
+        List<Row> expected =
+                Collections.singletonList(
+                        Row.ofKind(
+                                RowKind.INSERT,
+                                Byte.parseByte("127"),
+                                Short.parseShort("255"),
+                                Short.parseShort("32767"),
+                                Integer.parseInt("65535"),
+                                Integer.parseInt("2147483647"),
+                                Long.parseLong("4294967295"),
+                                Long.parseLong("9223372036854775807"),
+                                new BigDecimal("18446744073709551615")));
+        assertEquals(expected, result);
```

Comment:
Please use only assertj assertions, here and throughout the rest of this test class